src/test/java/com/noticemedan/cinema/mainTest.java

```diff
@@ -0,0 +1,7 @@
+package com.noticemedan.cinema;
+
+public class mainTest {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2187)